include/types.h

```diff
@@ -2020,6 +2021,7 @@ typedef struct user_options
   u32          debug_mode;
   u32          hwmon_temp_abort;
   int          hash_mode;
+  int          guess_hash_mode;
```

Comment:
Why not `bool` like `benchmark`?

```suggestion
  bool         guess_hash_mode;
```